.eslintignore

```diff
@@ -0,0 +1 @@
+packages/cryptoassets/data
```

Comment:
:rage1:

Reply:
yeah it's generated data, it's valid javascript. but eslint don't like `[,,,,stuff]` which is handy to save bytes.

Reply:
i was referring to the "missing EOL char" ^^ not so important though. usually it's better for git diff